packages/seo-integration/src/analyze.js

```diff
@@ -0,0 +1,144 @@
+import { Paper } from "yoastseo";
+
+/**
+ * Creates a paper based on the given data, keyphrase and configuration.
+ *
+ * @param {Object} data The paper data and metadata, e.g. content, title, date, slug.
+ * @param {Object} keyphrase The keyphrase to use for the analysis.
+ * @param {Object} configuration Extra configuration.
+ *
+ * @returns {Paper} A paper that can be analyzed using the web worker.
+ */
+function createPaper( data, keyphrase, configuration ) {
```

Comment:
Technically this could also be called a transformer I think, but This makes sense too 🙂